config/sidekiq.yml

```diff
@@ -1,2 +1,8 @@
 ---
 :concurrency: 1
+:logfile: ./log/sidekiq.json.log
+:timeout: 4
+:max_retries: 9
+:queues:
+  - [collect_postcodes, 4]
+  - [process_postcode, 2]
```

Comment:
I think we can remove these and just use `- default` - see https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues

Setting queue priorities is useful when we're bulk-processing lots of different workers at the same time, but we're planning to only ever have 2 active workers: one long-lived `PostcodesCollectionWorker`, and many many short-lived `ProcessPostcodeWorker` (one at a time).